develop/tutorials/articles/02-tooling/06-liferay-sample-modules/00-intro.markdown

```diff
@@ -0,0 +1,75 @@
+# Liferay Sample Modules
+
+Liferay provides working examples of sample modules that target different
+integration points in @product@. These working examples can be copy/pasted into
+your own independent project so you can take advantage of various Liferay
+extension points. Most of the sample modules include a single class using
+Declarative Services. Therefore, each sample is developed to demonstrate a
+single extension point and includes its own build files. Liferay's sample
+modules can be found in the
+[liferay-blade-samples](https://github.com/liferay/liferay-blade-samples)
+repository on Github.
+
+If you'd like to browse the repo locally or copy sample projects into your own
+project, [fork](https://help.github.com/articles/fork-a-repo/) and
+[clone](https://help.github.com/articles/cloning-a-repository/) the
+`liferay-blade-samples` repository.
+
+At first glance, you'll notice that the repository is broken up into four
+primary folders:
+
+- `bndtools`
+- `gradle`
+- `liferay-gradle`
+- `maven`
+
+The provided sample modules are organized into several different build chains to
+cater to a variety of developers. Each folder offers a set of Liferay projects
+that can be bootstrapped to that specific development environment. For example,
+the sample project residing in the `gradle` folder can be bootstrapped to a
+Gradle based development environment. The same concept also applies to the
+`bndtools` and `maven` projects. The `liferay-gradle` folder provides projects
+that are bootstrapped on to the Liferay Gradle plugin (e.g.,
+`com.liferay.plugin`). The Liferay Gradle plugin is really a set of Gradle
+plugins that makes developing Liferay applications quick and easy.
+
++$$$
+
+**Note:** There are two additional projects in the `/bndtools` folder that
+you'll need to build and publish modules:
+
+- `/bndtools/cnf` - the Bndtools configuration project.
+- `/bndtools/blade.run` - A Bndtools project which can push modules into an OSGi
+container and run them. The `biz.aQute.remote.agent-X.X.X.jar` must be deployed
+before this project can work properly.
+
+$$$
+
+The sample projects also demonstrate how to use various frameworks like:
+
+- Blueprint
+- Declarative Services (DS)
+- OSGi API
+
++$$$
+
+**Note:** Liferay does not provide a Blueprint implementation out-of-the-box. To
```

Comment:
Lets remove all talk of blueprint and apache aries... it is not needed for most of these samples.